### PR TITLE
translate definitions-time into zh-hans

### PR DESCRIPTION
**Pull request by cuihuan**
https://www.w3.org/International/articles/definitions-time/
https://www.w3.org/International/articles/definitions-time/index.zh-hans

### CODE REVIEW

**Comment by r12a** [comment]
Is this still WIP ?

**Comment by cuihuan** [comment]
> Is this still WIP ?

Yes, I'll correct the details later.

**Comment by cuihuan** [comment]
@xfq Translation completed, thank you for CR
In addition, do we have a i18n or time vocabulary?

**Comment by r12a** [comment]
@xfq could you review this translation? Once it passes, i'll update the tracker and announce the 2 zh-hans translations we have pending.

File: articles/definitions-time/index.zh-hans.html [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+</header>
+
+
+<section>
+  <div id="audience">
+    <div id="updateInfo"></div><script>document.getElementById('updateInfo').innerHTML = g.updated</script>
+    </div>
+
+    <p>本文主要介绍一些基本的术语和概念，有关于时区的处理以及Web上日期和时间的处理，这些基本概念在您阅读其他文章时也会遇到。</p>
+    <p><a href="https://www.w3.org/TR/timezone/">《时区使用》</a>这篇文档中，提供了这些问题更多的细节信息，以及web上处理时间和日期的一些规范，深入阅读可以参考此文。</p>
```
Comment (xfq) [inline articles/definitions-time/index.zh-hans.html]:
```suggestion
    <p><a href="https://www.w3.org/TR/timezone/">《时区使用》</a>这篇文档中，提供了这些问题更多的细节信息，以及Web上处理时间和日期的一些规范，深入阅读可以参考此文。</p>
```

“Web”的大小写建议和原文一致。

File: articles/definitions-time/index.zh-hans.html [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+  <p class="info"><dfn id="def_observed_time">观测时间</dfn>是指一种依赖于物理观察来确定时间的方式，可以依赖于太阳，月亮或者星星等物理事物，不过当前基本废用了。观测时间里的中午是指太阳在空中的最高点，但对于仅仅相距几公里的人来说，实际的中午时间可能会有所不同。</p>
+  </aside></div>
+  </section>
+
+
+  <section id="floating_time">
+  <h2>浮选时间</h2>
+  <p>有些时间值并不能映射到<a href="incremental_time" class="termref">累计时间</a>中的某个特定时刻。与之相反，它们仅仅代表一个名义时间，在世界各地的所有时区都会以相同的方式呈现。</p>
+  <p>举个例子，2016年6月11日星期六是英国女王官方的90岁生日。由于不同国家的时钟设置差异，6月11日在英国开始或结束的具体时间，在其他国家实际上有可能是周五或者周日了，但该事件的日期始终是6月11日星期六。</p>
+  <p>由于没有对应明确的<a href="#incremental_time" class="termref">累计时间</a>，因此，我们通常称此类时间为<dfn id="def_floating_time">浮选时间</dfn>。当绑定上对应的时区信息后，浮选时间会生成<em>一系列可被接收的累计时间的值</em>。</p>
```
Comment (xfq) [inline articles/definitions-time/index.zh-hans.html]:
```suggestion
  <p>由于没有对应明确的<a href="#incremental_time" class="termref">累计时间</a>，因此，我们通常称此类时间为<dfn id="def_floating_time">浮选时间</dfn>。当绑定上对应的时区信息后，浮选时间会生成<strong>一系列可被接收的累计时间的值</strong>。</p>
```

中文传统上不使用斜体引起读者注意，建议改为`<strong>`。

File: articles/definitions-time/index.zh-hans.html [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+  <p>浮选时间不会也不应该归属于某个特定的时区。</p>
+  <p>举一些其他浮选时间事件的例子，比如：报纸的出版日期、里约奥运会的开始日期、新年开始的时间、办公时间设置为“9点到5点”等等，均不需要考虑<a href="#time_zone">时区</a>。</p>
+  </section>
+
+
+<section id="time_zone">
+  <h2>时区</h2>
+  <p><dfn id="def_time_zone">时区</dfn>是一组用于确定本地时间（也叫时钟时间）的规则，它与特定地理区域的累计时间（在大多数的计算系统中使用）相关。</p>
+  <p>如果要关联到时钟时间，时区规则中必须要考虑<a href="#time_zone_offset" class="termref">时区偏移量</a><em>和</em>任何<a href="#dst" class="termref">夏令时</a>的变更。</p>
```
Comment (xfq) [inline articles/definitions-time/index.zh-hans.html]:
```suggestion
  <p>如果要关联到时钟时间，时区规则中必须要考虑<a href="#time_zone_offset" class="termref">时区偏移量</a><strong>和</strong>任何<a href="#dst" class="termref">夏令时</a>的变更。</p>
```

中文传统上不使用斜体引起读者注意，建议改为`<strong>`。

File: articles/definitions-time/index.zh-hans.html [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+  <p>在TZ database中，时区的ID通常由地区和示例城市组成，示例城市是相关时区中人们所熟知的典型城市。比如，美国太平洋时区在TZ database中的ID是<code class="kw" translate="no">America/Los_Angeles</code>。同时，TZ database还为许多ID提供了别名， 比如<code class="kw" translate="no">Asia/Ulan Bator</code>也可以称为<code class="kw" translate="no">Asia/Ulaanbaatar</code>.</p>
+  <p>CLDR（通用本地数据集 Common Locale Data Repository的英文首字母缩写）可以为ID提供本地化的形式。需要注意，部分系统（比如苹果的Mac OS）会提供额外的示例城市。</p>
+  </section>
+
+
+  <section id="calendars">
+  <h2>公历和其他日历</h2>
+  <p><dfn id="def_gregorian_calendar">公历</dfn>是使用最广泛的历法。它是太阳历, 一年通常365天，包含“闰年”的概念。每4年为闰年，增加一天，但当年份可以被100整除时，不用增加（同时，如果年份可以被400整除，仍需要增加）。</p>
+
+  <p>世界各地还有许多其他历法。其中一部分是月亮历；还有一部分基于公历，但开始时间不一样；还有一些历法随着某些重要人物的死亡而重置。通常这些历法用于宗教目的，但有时也被用于报纸和电子邮件，也可以是出生日期。</p>
```
Comment (xfq) [inline articles/definitions-time/index.zh-hans.html]:
```suggestion
  <p>世界各地还有许多其他历法。其中一部分是阴历；还有一部分基于公历，但开始时间不一样；还有一些历法随着某些重要人物的死亡而重置。通常这些历法用于宗教目的，但有时也被用于报纸和电子邮件，也可以是出生日期。</p>
```

lunar calendar一般译为[阴历](https://zh.wikipedia.org/wiki/%E9%98%B4%E5%8E%86)。

File: articles/definitions-time/index.zh-hans.html [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+  <p>ICU或Dojo等技术可以支持不同日历之间的转换。</p>
+
+  </section>
+
+
+  <section id="html5_terms">
+  <h2>HTML5 条款</h2>
```
Comment (xfq) [inline articles/definitions-time/index.zh-hans.html]:
```suggestion
  <h2>HTML5术语</h2>
```

term此处意思是“术语”，下面提到的都是HTML标准中的术语，如[global date and time](https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#concept-datetime)。

另外，建议汉字与西文字母/数字之间统一**不添加空格**（和目前的[W3C中文排版需求](https://github.com/w3c/clreq/)一致）。

File: articles/definitions-time/index.zh-hans.html [hunk 6]
```diff
@@ -0,0 +1,218 @@
...
+<ul id="full-links">
+  <li><p><cite><a href="https://www.w3.org/TR/timezone/">时区使用</a></cite></p></li>
+  <li>
+    <p><cite>HTML&CSS创作</cite></p>
+    <ul>
+      <li><a href="/International/techniques/authoring-html#localdata">日期格式化</a></li>
+      </ul>
+  </li>
+  <li>
+    <p><cite>开发指南</cite></p>
```
Comment (xfq) [inline articles/definitions-time/index.zh-hans.html]:
```suggestion
    <p><cite>标准开发指南</cite></p>
```

**Comment by xfq** [comment]
> do we have a i18n or time vocabulary?

We have https://www.w3.org/TR/i18n-glossary/ , and if you think there are terms that need to be added, feel free to file issues in https://github.com/w3c/i18n-glossary/issues/ . We also welcome translations of the document.

**Comment by cuihuan** [comment]
> > do we have a i18n or time vocabulary?
> 
> We have https://www.w3.org/TR/i18n-glossary/ , and if you think there are terms that need to be added, feel free to file issues in https://github.com/w3c/i18n-glossary/issues/ . We also welcome translations of the document.

thanks , i will try to translate this document.

**Review by xfq — APPROVED** [review]

**Comment by r12a** [comment]
Merged. Thank you.  (Note: there is at least one small divergeance from the new template format, so i will update the translation to fix that this morning.)

**Pull request merged**